images/nginx/matomo/80-nginx-matomo-config

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# Inject Matomo tracking snippets via nginx configuration.
+
+# Single site configuration.
+if [ ! -z "$MATOMO_URL" ]; then
+
+    # Add Matomo tracking code to end of the HEAD section.
+    cat <<EOF >> /etc/nginx/conf.d/matomo.conf
+sub_filter '</head>'
+"<!-- Matomo -->
+<script type='text/javascript'>
+var _paq = window._paq = window._paq || [];
+/* tracker methods like 'setCustomDimension' should be called before 'trackPageView' */
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+(function() {
+    var u='${MATOMO_URL}';
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', '${MATOMO_SITE_ID:-1}']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
+})();
+</script>
+<!-- End Matomo Code -->
+</head>";
+sub_filter_once on;
+EOF
+
+    fix-permissions /etc/nginx/conf.d/
```

Comment:
we don't need fix-permissions in entrypoints scripts, as in the context of an entrypoint the user is already the random user id. fix-permissions is really only useful during a Docker build as the docker build is running as `root` and therefore would create files with inaccessible permissions